code/datums/quirks/good.dm

```diff
@@ -202,6 +357,52 @@
 	give_item_to_holder(/obj/item/storage/fancy/candle_box, list(LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 	give_item_to_holder(/obj/item/storage/box/matches, list(LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 
+/datum/quirk/item_quirk/super_immune
+	name = "Super Immune"
+	desc = "You have a robust immune system."
+	icon = "pills"
+	value = 4
+	mob_trait = TRAIT_SUPER_IMMUNE
+	medical_record_text = "Patient has a hightened immune system."
+
+/datum/quirk/item_quirk/sweet_tooth
+	name = "Sweet Tooth"
+	desc = "You have a thing for sweets, they make you feel happy."
+	icon = "cookie"
+	value = 2
+	mob_trait = TRAIT_SWEET_TOOTH
+	mood_quirk = TRUE
+	gain_text = "<span class='notice'>You want somthing sweet.</span>"
+	lose_text = "<span class='danger'>You loose your drive for something sweet.</span>"
+	medical_record_text = "Patient has poor dental enamel."
+
+/datum/quirk/item_quirk/thick_skin
+	name = "Thick Skin"
+	desc = "Your skin is tough as leather, leaving you resistant to blunt and slashing wounds."
+	icon = "user-shield"
+	value = 6
+	mob_trait = TRAIT_THICK_SKIN
+	medical_record_text = "Patient has abnormally thick skin."
+
+/datum/quirk/robust_physique
+	name = "Built From Stronger Stuff"
+	desc = "You have robust physique, allowing you to take more punishment."
+	icon = "user-astronaut"
+	value = 10
+	medical_record_text = "Patient has an unnaturally robust physique."
+
+/datum/quirk/robust_physique/add()
+	if(iscarbon(quirk_holder))
+		var/mob/living/tough_guy = quirk_holder
+		tough_guy.maxHealth *= 1.2
```

Comment:
I suspect this doesn't necessarily change crit thresholds but I might be wrong.

Reply:
it doesnt change crit thresholds, those are hardcoded values

Reply:
That's fucking stupid.

Reply:
I thought it did in testing. I'll try again